requirements-tests-py3.txt

```diff
@@ -1,6 +1,6 @@
 mypy==0.910
 pytype==2021.11.2
-typed-ast==1.4.3
+typed-ast==1.5.0
```

Comment:
I'm surprised this works, as mypy 0.910 is incompatible with typed-ast 1.5.0. In fact, when I try to pip install this in a 3.6 venv, pip fails. Maybe we should just remove the typed-ast dependency here?

Reply:
This works on Python 3.8+ because the typed-ast dependency is an optional extra that we only need for parsing Python 2 syntax (and stub files are never treated as Python 2, see https://github.com/python/mypy/blob/aeb65a6ff8cace69d4f2a04dcf511cce2cd8bf21/mypy/parse.py#L23).

Reply:
That is, I'm pretty sure this is a totally legitimate fix :-)

Reply:
The problem is that this change means that you can't run typeshed tests with Python 3.6 or 3.7 (as there is now a conflict between requirements and mypy's dependencies when running on < 3.8). That's fine by me, but we should at least make a note in `CONTRIBUTING`.

Reply:
Also, do we need the explicit typed-ast dependency? I just removed it locally, and the tests seem to run fine with Python 3.9. (pytype brought in typed-ast 1.5.0.) So, I suggest we do both: Add a note to `CONTRIBUTING.md` and remove `tests/README.md` that tests require Python 3.8+ and remove the dependency here.